spotify_player/src/cli/client.rs

```diff
@@ -344,9 +365,435 @@ async fn handle_playback_request(
             };
             PlayerRequest::SeekTrack(progress + chrono::Duration::milliseconds(position_offset_ms))
         }
+        _ => unreachable!(),
```

Comment:
I don't think `unreachable!` is needed here. `unreachable!` should be avoided when matching `enum` as it defeats the purpose of enum exhaustive checking.

Reply:
I agree, but I don't think it will compile without. Doesn't each 'handle' only match part of the Command enum?

Reply:
Nevermind, implemented the PlaylistCommand enum so now it doesn't need it.